### ISSUE TITLE
Unable to load data: error getting cached app managed resources

### ISSUE THREAD

**Issue by da6d6i7-bronga** [argoproj/argo-cd#13241]
Checklist:

* [X ] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [X ] I've included steps to reproduce the bug.
* [ ] I've pasted the output of `argocd version`.

**Describe the bug**

After doing a clean deployment of ArgoCD HA v2.6.7 and navigating to an application tile of a test app I fed to ArgoCD I see the deployment tree does not fill out completely and opening a 'kind' tile will fire off the error:

`Unable to load data: error getting cached app managed resources: dial tcp: lookup argocd-redis on 10.119.252.10:53: no such host`

The test application deployment itself is successful. Also, the Pod and Network pages are empty. A team mate pointed out that argocd-redis is the non-HA name of the service, which does not exist, shouldn't it be argocd-redis-ha* ?

This is not happening with our non-HA deploys

**To Reproduce**

Just deploy HA v2.6.7

**Expected behavior**

Should behave the same as the non-HA v2.6.7

**Screenshots**

![image](https://user-images.githubusercontent.com/31454345/232132588-81f3676f-4cea-415d-84dc-9acceb14c116.png)

**Comment by justinwalz** [comment]
I have experienced a similar issue. Without doing too much debugging, my first guess was to check the associated network policy. As a test, I removed it on a non-prod environment and I no longer saw this error on the UI.

Perhaps the selectors are not correct?

**Comment by dan-vystrcil** [comment]
@justinwalz you are correct. I removed the network policies and it started working properly.

**Comment by rishikanthc** [comment]
@dan-vystrcil could you tell me which network policies were causing the issue ? I'm also facing the same issue and i'm not sure which one to delete. I tried removing the argocd-redis-ha-server-network-policy and the UI stopped loading

**Comment by dvystrcil** [comment]
Hi @rishikanthc I ended up removing all the network policies.
